extensions/wikia/CommunityPage/scripts/ext.communityPage.js

```diff
@@ -124,16 +125,16 @@ require([
 				}
 			};
 			uiModal.createComponent(createPageModalConfig, function (modal) {
-				var loading = mustache.render(templates.modalLoadingScreen, {
-						loadingText: $.msg('communitypage-modal-tab-loading'),
-					}),
+				var loading = mustache.render(templates.modalLoadingScreen),
 					html = navHtml + loading;
```

Comment:
We can remove loading and html variables and construct when calling html() as it's used only there.